### ISSUE TITLE
While using NVDA/Narratorr, the Ctrl + Alt + Arrow key navigation shortcuts are not working in the table: A11y_Azure Databases extension for VS Code_Info and relationship.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/vscode-cosmosdb#2929]
### GitHub Tags:
#A11ySev2; #A11yTCS; #DesktopApp; #BM_Azure Databases extension for VS Code_Win32_Nov2025; #Win32; #Win11; #FTP; #Azure Databases extension for VS Code; #Benchmark; #AILimited; #MAS1.3.1; #Info and Relationship; #Element:text; #Screen Reader; #NVDA; #Narrator; #STP; #A11yMAS;

"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification."

"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details:
Application Name: Azure Databases extension for VS Code
OS Version: Windows 11 enterprise 24H2. (OS build 26120.3360)
Screen Reader: Narrator, NVDA (2025.3.2)

### Steps to Reproduce

1. Turn on NVDA/Narrator.
2. Open VS Code
3. Open the "Azure" Sidebar
4. Sign In to your Azure Account (skip if already signed in)
5. Expand the "Cosmosdb-Portalteam-generaltest-msft" subscription node
6. Expand the "Azure Cosmos DB" node
7. Expand the "a11y-nosql" Cosmos DB account
8. Expand the "SampleDB" database node
9. Expand the "SampleContainer" node.
10. Tab till "Create query" and press enter.
11. Run a simple query such as "SELECT TOP 1000 * FROM c"
12. Tab till "Results" tab and press enter.
13. Tab till "Table" 
14. Observed the issue While using NVDA, the Ctrl + Alt + Arrow key navigation shortcuts are not working in the table.

### Actual Results:
While using NVDA/Narrator, the Ctrl + Alt + Arrow key navigation shortcuts are not working in the table.

### Expected Results:
Ctrl + Alt + Arrow key NVDA/Narrator navigation shortcuts should work consistently within the table.

### User Impact:
Screen reader users cannot navigate efficiently using NVDA’s/Narrator structural navigation. They must rely on less efficient methods, slowing down reading and editing workflows. This reduces accessibility and usability for blind or low-vision users.

### Attachment:

<img width="1365" height="716" alt="Image" src="https://github.com/user-attachments/assets/fd5637f3-e746-465e-9c22-abf3ed67a1a4" />

[2929_A11y_Azure Databases extension for VS Code_Narrator.zip](https://github.com/user-attachments/files/25068742/2929_A11y_Azure.Databases.extension.for.VS.Code_Narrator.zip)

[2929_A11y_Azure Databases extension for VS Code_NVDA.zip](https://github.com/user-attachments/files/25068764/2929_A11y_Azure.Databases.extension.for.VS.Code_NVDA.zip)

**Comment by bk201-** [comment]
Hi @PadminiGariki! 
I've tried to find the guide or doc where this explains, but I didn't have luck. There is already navigation by arrow keys, but what is the reason to also support another key combination?  Moreover, even fluent UI grid does not support this combination

https://storybooks.fluentui.dev/react/?path=/docs/components-datagrid--docs#focusable-elements-in-cells

**Comment by sevoku** [comment]
@PadminiGariki this doesn't look like a Sev2 to me since there is an alternative using just the Arrow Keys (without <kbd>Ctrl</kbd>+<kbd>Alt</kbd>) as @bk201- pointed out. Can you please lower the severity and let's double check if this is a usability issue in this scenario at all.
